tests/acceptance/ColumnsTasksCept.php

```diff
@@ -42,5 +42,5 @@
 $I->see( 'Columns Block description', 'div' );
 $I->see( 'Column 1', '.step-info h5' );
 $I->see( 'Column 1 description', '.step-info p' );
-$I->seeElement( '.steps-action img' );
+$I->seeElement( '.steps-action img, .steps-info img' );
```

Comment:
The class name changes from [parent PR](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/pull/342/files#diff-f7fead8b4dcda09f96e0342aa00aec27R21) is `step-info`, if I am not wrong it would be `step-info` right?

Reply:
🤦